Translation corrections (#418)
07th-mod/higurashi-console-arcs: Update/ura_004.txt

//rそれでも…知り合ったばかりの頃は、待ち合わせた店で何も注文せずにお冷だけを飲んでいたことを考えれば、少しは遠慮のない関係に近づけたのかもしれない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それでも…知り合ったばかりの頃は、待ち合わせた店で何も注文せずにお冷だけを飲んでいたことを考えれば、少しは遠慮のない関係に近づけたのかもしれない。",

> 			NULL, "Still... compared to when they first met, it seemed like she was feeling a little less reserved around her, even if she didn't order anything other than a cold drink.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Still... compared to when they first met, when she didn't order anything at their meetings and just drank water, it seemed like she was feeling a little less reserved.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>客</color>", NULL, "<color=#f5e6d3>Customer</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_cusa1001", 256, TRUE);
	OutputLine(NULL, "『…そもそも赤いなんとかってやつに、ロクなやつがいたためしがない』",

> 			NULL, "\"...In the first place, I've never met a red-something-or-other who turned out to be any good.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...In the first place, I've never met someone dressed in red who turned out to be any good.\"", Line_Normal);
```

> 			NULL, "\"...In the first place, I've never met someone dressed in red who turned out to be any good.\"", Line_Normal);

Reviewer 2: The red here was the color of the marker they labeled the image with

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>客</color>", NULL, "<color=#f5e6d3>Customer</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_cusb1001", 256, TRUE);
	OutputLine(NULL, "『ブラックホールに引き込まれる隕石とか小惑星とかって、きっとああいう風に消えていくんだろうな…』",

> 			NULL, "\"I'm sure that's what a black hole looks like when it's devouring asteroids and planets...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"I'm sure that's what a black hole looks like as it's devouring asteroids and planets...\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s14/28/vt4bb_tomoe1106", 256, TRUE);
	OutputLine(NULL, "（それが、この子の根っこをダメにしているのね…）",

> 			NULL, "(So, that corrupted the core of her being, huh...)", Line_Normal);

Reviewer: ```suggestion
			NULL, "(So, that corrupted her down to her core, huh...)", Line_Normal);
```

//rだけど、と巴は首を振る。…このままだと夏美は、ずっとこの悩みを持ち続けたままじゃないか。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "だけど、と巴は首を振る。…このままだと夏美は、ずっとこの悩みを持ち続けたままじゃないか。",
			NULL, "But, Tomoe shook her head. ...At this rate, Natsumi would continue to carry these worries for a long time coming.", Line_Normal);
	ClearMessage();

//巴r-vS14/28/VT4Bb_tomoe1107.（…それで、いいわけがない）
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s14/28/vt4bb_tomoe1107", 256, TRUE);
	OutputLine(NULL, "（…それで、いいわけがない）",
			NULL, "(...I can't just leave it at that.)", Line_Normal);
	ClearMessage();

//r…わかっている。お節介かもしれないと、誰に言われるまでもなく理解はしている。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…わかっている。お節介かもしれないと、誰に言われるまでもなく理解はしている。",

> 			NULL, "...She didn't need anyone telling her to know that might be considered meddlesome.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...She didn't need anyone saying anything to know that might be considered meddlesome.", Line_Normal);
```
